docs/faq.rst

```diff
@@ -167,7 +167,141 @@ The maximum parallelism depends on the number of partitions.
          against a two-partition input topic, only two servers perform the actual work, but the other eight will run an “idle”
          query.
 
+=================================================================================
+Can KSQL connect to an Apache Kafka cluster over SSL and authenticate using SASL?
+=================================================================================
+
+Yes. Internally, KSQL uses standard Kafka consumers and producers.
+The procedure to securely connect KSQL to Kafka is the same as connecting any app to Kafka.
+
+For example, you can add the following entries to the KSQL server configuration file
+(ksqlserver.properties). This configuration enables KSQL to connect to a secure Kafka
```

Comment:
this file is now called `ksql-server.properties`? cc @hjafarpour

Reply:
Done. 

(Though TBH the file can be called anything the user likes, right? It just happens that the stock one we supply is called that).

Reply:
@big-andy-coates : Yes, you are right.  But what typically happens is that, in trainings/questions/etc many users will stick to whatever the default name is (and also copy-paste instructions verbatim).